dephell/converters/wheel.py

```diff
@@ -128,7 +128,9 @@ def dump(self, reqs, path: Path, project: RootDependency) -> None:
                     for full_path in package:
                         self._write_file(
                             archive=archive,
-                            path='/'.join(full_path.relative_to(project.package.path).parts),
+                            path='/'.join(
+                                full_path.relative_to(project.package.path.joinpath(
+                                    project.package.package_dir.get('',""))).parts),
```

Comment:
Sorry, it still doesn't work. `.get` return an empty string if no match found, and it fails:

```python
from pathlib import Path
Path().absolute().relative_to('')
# ValueError: '/home/gram' does not start with ''
```

Please, include a few test cases. This should help to see what is possible:
https://github.com/dephell/dephell_discover/blob/master/tests/test_root.py#L42

Thank you for taking care of this. I've done with most of my other projects, so now I'll try to give you feedback faster :)

Reply:
I think I figured out the reason for that. I set the default value to "" (empty) instead of the current directory ".".
I rewrote the code to check if the dictionary is empty and if not, it takes the first value of the dictionary. This is usually only one entry.

I must confess, I don't where I can write those testcases. You wrote all testcases very generic in the test_all.py file. All other converters only implement testcases for loading. 
I try to stick to your scheme of implementing those tests, but at the moment, it is a little bit overwhelming.
Could you give me a hint or an example how to implement the dump test for the wheel package?

Reply:
Taken from your next comment:

"Using the current dir as default is not correct."

If the local folder is not the best default value for that, what would you consider a good alternative?
I think, if the project folder was set for another path, that path should be in that structure nevertheless.

Falling back to the value `project.package.path` might be a safe choice for that situation.

Reply:
Figured out another approach, which corresponds with how `setuptools` is generating the wheel package.
The exact approach is described in the commit messages and in the source code itself.